corehq/apps/userreports/reports/view.py

```diff
@@ -112,19 +112,42 @@ def get_context_data(self, **kwargs):
 
     @property
     def saved_report_context_data(self):
+        def _update_daterange_filters(report_config):
+            report_config_data = ReportConfig.get_db().open_doc(report_config._id)
```

Comment:
Is the goal of this line to get the document as a dictionary instead of a `ReportConfig`? There is probably a way to "unwrap" the object that wouldn't require a db read.